modules/ROOT/pages/policies-compare-mule3-and-mule4.adoc

```diff
@@ -0,0 +1,41 @@
+= Policies and Mule Versions
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: policy, custom, ootb, offline
+
+In Mule versions 3 and 4, the policies remain the same, with a few exceptions. The following section describes the similarities and differences in these Mule versions.
+
+[%header,cols="40a,50a,50a"]
+|===
+| Feature | Mule 3 | Mule 4 
+|           | Uses MEL expressions | Uses DataWeave 2.0 
```

Comment:
The name of the feature is missing, it should be "Expression Language"

Reply:
Done! Thanks!